readme/vaccination.md

```diff
@@ -0,0 +1,34 @@
+## Vaccination
+This report searches different vaccinations. It is a MSPP Report requested by the Haitian MoH to any clinic in Haiti.
+
+**Setup**
+
+It is setup by setting the following properties in [initializer](https://github.com/mekomsolutions/openmrs-module-initializer) `jsonkeyvalues`' domain configuration. 
+
+```bash
+{
+    ...
+    ...,
+        "report.vaccination.active": "true",
+        "report.ecv.active": "true",
+        "report.vaccination.prenatalVisitType": "uuid-to-prenatal-visitType",
+        "report.vaccination.vaccinations": "uuid-to-vaccinations-question-concept",
+        "report.vaccination.vaccinationSequenceNumberConcept": "uuid-to-vaccination-sequence-number-concept",
+        "report.vaccination.boostervaccinationSequenceNumberConcept": "uuid-to-vaccination-booster-number-concept",
```

Comment:
Rename to `boosterSequenceNumberConcept`

Reply:
Done